bot.user.js

```diff
@@ -370,13 +377,24 @@ window.onFrameUpdate = function() {
         }
     }
 };
+window.playDefence = function(dir) {
+    window.kd_l = (dir === "l");
+    window.kd_r = (dir === "r");
```

Comment:
This, as well as [this](https://github.com/ErmiyaEskandary/Slither.io-bot/pull/47/files#diff-3deb4bbb2131417b70ba135e5a6f7715R381) simulate the pressing of the left/right arrows, to turn in the smallest radius circle.